content/zh/docs/ops/diagnostic-tools/cni/index.md

```diff
@@ -38,9 +38,9 @@ CNI DaemonSet 的就绪表明 Istio CNI 插件已正确安装和配置。
 `istio-cni-node` 守护进程日志进行诊断。您还可以通过 `istio_cni_install_ready`
 指标跟踪 CNI 安装准备情况。
 
-## Race 条件修复 {#race-condition-repair}
+## 竞态条件修复 {#race-condition-repair}
```

Comment:
```suggestion
## 竞争条件和缓解措施 {#race-condition-repair}
```

Reply:
"race-condition-repair" here should refer to measures taken to mitigate race conditions. Using the word "和" seems not very appropriate.

Reply:
In a normal case, it's a good practice to keep a link text consistent with the heading [after clicked](https://istio.io/latest/zh/docs/setup/additional-setup/cni/#race-condition-mitigation): 

Also, this link text has some variants in this article:

- race condition repair
- race condition mitigation

If zh text provides a consistent expression, it will be helpful to enhance readability IMO.